samples/cpp/text2image/README.md

```diff
@@ -45,6 +45,7 @@ Models can be downloaded from [HiggingFace](https://huggingface.co/models). This
 - [stabilityai/stable-diffusion-2-1](https://huggingface.co/stabilityai/stable-diffusion-2-1)
 - [dreamlike-art/dreamlike-anime-1.0](https://huggingface.co/dreamlike-art/dreamlike-anime-1.0)
 - [SimianLuo/LCM_Dreamshaper_v7](https://huggingface.co/SimianLuo/LCM_Dreamshaper_v7)
+- [stabilityai/stable-diffusion-xl-base-1.0](https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0)
```

Comment:
There's a dedicated file for supported models https://github.com/openvinotoolkit/openvino.genai/blob/master/src/docs/SUPPORTED_MODELS.md because not everyone likes the idea of listing the models in samples. Would you move this list in one of the following PRs?

Reply:
yes, it was one of TODOs for text 2 image pipeline